lib/project_types/script/layers/infrastructure/languages/assemblyscript_task_runner.rb

```diff
@@ -31,12 +43,23 @@ def metadata_file_location
           end
 
           def library_version(library_name)
+            ensure_environment
```

Comment:
If you wanted, you could move the ensure_environment check in a new `npm_command` method which would take care of validating that the env is valid (once) and then running the command. 🤷